Makefile

```diff
@@ -11,5 +11,6 @@ theme: mtheme
 
 clean:
 	latexmk -c
+	rm -rf _minted-slides/ *.sty *.snm *.vrb *.nav
```

Comment:
The .sty files are from the theme (see `Makefile`). They aren't regenerated when you build the presentation.